oidc-controller/api/core/oidc/issue_token_service.py

```diff
@@ -98,7 +98,14 @@ def get_claims(
 
         if sub_id_claim:
             # add sub and append presentation_claims
-            oidc_claims.append(Claim(type="sub", value=sub_id_claim.value))
+            assert type(auth_session.request_parameters["pres_req_conf_id"]) == str
+            oidc_claims.append(
+                Claim(
+                    type="sub",
+                    value=f"{sub_id_claim.value}@{auth_session.request_parameters['pres_req_conf_id']}",
+                )
+            )
+
         elif ver_config.generate_consistent_identifier:
```

Comment:
You're going to want to append value of the `pres_req_conf_id` also when generating a consistend `sub` as the generated value will depend on the proof-request configuration.

In that case, you might want to save the value of `pres_req_conf_id` before the `if/else` so you can use it in both places without having to explicitly access the attribute in two spots.

Reply:
If I understand this correctly this issue should now be resolved